ggircs-app/app/components/LoginButton.tsx

```diff
@@ -12,21 +11,25 @@ const LoginButton: React.FunctionComponent = () => {
     )}`;
 
   return (
-    <Form inline action={loginURI} method="post">
+    <form action={loginURI} method="post">
       <button type="submit">Log in</button>
       <style jsx>{`
-        button {
-          color: #f8f9fa;
-          display: inline-block;
-          text-align: center;
-          user-select: none;
-          background-color: transparent;
-          border: 1px solid #f8f9fa;
-          padding: 0.375rem 0.75rem;
-          font-size: 1rem;
-          border-radius: 0.25rem;
+        button[type="submit"] {
+          background: none !important;
+          border: none;
+          display: flex;
+          font-size: 0.813em;
+          font-weight: normal;
+          color: #fff;
+          padding: 0 15px 0 5px;
+          text-decoration: none;
+          border-right: 1px solid #9b9b9b;
+        }
+        button[type="submit"]:hover {
+          text-decoration: underline;
+        }
```

Comment:
Is the [button component](https://button-inc.github.io/service-development-toolkit/@button-inc/bcgov-theme/index.html?path=/docs/components-button--default) not good here?

Reply:
It kind of breaks the style of https://button-inc.github.io/service-development-toolkit/@button-inc/bcgov-theme/index.html?path=/docs/components-navigation--default

Reply:
Ok, maybe a feature request for a `nav` variant, or compatibility with the navigation component out of the box would be good